### PR TITLE
Jahnavi/contribute p0

### PR DESCRIPTION
**Pull request by JahnaviChowdary**

### CODE REVIEW

File: .NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseSetParser.cs [hunk 1]
```diff
@@ -111,10 +162,21 @@ public List<DateTimeParseResult> FilterResults(string query, List<DateTimeParseR
             return candidateResults;
         }
 
-        private DateTimeResolutionResult ResolveSet(ref DateTimeResolutionResult result, string innerTimex)
+        private DateTimeResolutionResult ResolveSet(ref DateTimeResolutionResult result, string innerTimex, DateTimeParseResult pr = null)
         {
             result.Timex = innerTimex;
             result.FutureValue = result.PastValue = "Set: " + innerTimex;
+
+            if (pr != null && pr.TimexStr.EndsWith("WE"))
```
Comment (tellarin) [inline .NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseSetParser.cs:170]: Again. Please also add a code comment explaining the change.

File: Specs/DateTime/Dutch/DateTimeModel.json [hunk 2]
```diff
@@ -1418,7 +1418,7 @@
             {
               "timex": "P1D",
               "type": "set",
-              "value": "not resolved"
+              "value": "2016-11-07"
```
Comment (tellarin) [inline Specs/DateTime/Dutch/DateTimeModel.json:1421]: This is incorrect. A set that recurs every day doesn't map to a single date.

File: Specs/DateTime/Dutch/SetParser.json [hunk 3]
```diff
@@ -60,10 +60,10 @@
         "Value": {
           "Timex": "P1D",
           "FutureResolution": {
-            "set": "Set: P1D"
+            "date": "2017-09-27"
```
Comment (tellarin) [inline Specs/DateTime/Dutch/SetParser.json:63]: Same as above and in other places.

File: Specs/DateTime/English/DateTimeModel.json [hunk 4]
```diff
@@ -16364,30 +16364,6 @@
       }
     ]
   },
-  {
-    "Input": "Let's make sure that happens every weekday",
```
Comment (tellarin) [inline Specs/DateTime/English/DateTimeModel.json:16368]: Why remove test spec cases? This should never happen.

File: Specs/DateTime/Portuguese/SetParser.json [hunk 5]
```diff
@@ -44,17 +44,20 @@
...
+    },
     "Results": [
       {
         "Text": "diariamente",
         "Type": "set",
         "Value": {
           "Timex": "P1D",
           "FutureResolution": {
-            "set": "Set: P1D"
+            "date": "2019-11-25"
```
Comment (tellarin) [inline Specs/DateTime/Portuguese/SetParser.json:57]: Incorrect resolution.

**Review by tellarin — CHANGES_REQUESTED** [review]
It seems this PR mixes a lot of things together. I suggest separating it into different PRs.
Firstly, please pay attention to PR titles as they'll be used as commit messages. Try to make them informative.
Second, if changing regexes to support new cases, please add new specs demostrating the new functionality.
Thirdly, when adding new funcionality, the PR should describe what it is. But it seems the changes here introduce incorrect behaviour that can't be merged.

File: Specs/DateTime/Dutch/DateTimeModel.json (hunk 2 above)
Comment (JahnaviChowdary) [reply Specs/DateTime/Dutch/DateTimeModel.json:1421]: We want to resolve to a date reference which is the next immediate valid recurrence date from the reference date

**Comment by JahnaviChowdary** [comment]
Closing this PR and will raise separate PRs

**Pull request closed without merging**